### PR TITLE
chore: use new dockerfile for building demo site

### PR DESCRIPTION
**Pull request by benjdlambert**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,20 +1,44 @@
...
-# Really annoying that you can't copy the tree easily using docker COPY command.
-# https://github.com/moby/moby/issues/15858#issuecomment-383690925
-ADD packages/backend/package.json ./packages/backend/package.json
-ADD packages/app/package.json ./packages/app/package.json
+# Uncomment this line if building a non create-app version
+# COPY packages packages
+COPY plugins plugins
 
-RUN yarn install --frozen-lockfile --production
+RUN find packages \! -name "package.json" -mindepth 2 -maxdepth 2 -print | xargs rm -rf
```
Comment (OrkoHunter) [inline Dockerfile:11]: I would appreciate a line or two as comment for this magic! 🙇 :)

Comment (benjdlambert) [reply Dockerfile:10]: Will address this in the main repo when we do this

Comment (benjdlambert) [reply Dockerfile:10]: This dockerfile will replace the main one for create-app etc

Comment (OrkoHunter) [reply Dockerfile:10]: Thanks!

**Pull request merged**